app/budget/directives/courseCosts/courseCostRow/courseCostRow.html

```diff
@@ -14,7 +14,6 @@
 		</div>
 		<div>
 			<input
-				type="number"
 				ng-model="sectionGroupCost.enrollment"
```

Comment:
by removing type="number" you are removing some of the 'validations' on the input fields, as well as getting rid of the little arrow button-y things on them I think.

I assume you wanted the arrow buttons to go away? If so I can make some inputs that properly validate for numbers only, without the arrows. Poke me when we're both in and let me know what you're envisioning.